docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -326,6 +326,13 @@ You can resolve this on Debian-based systems, for example, by running::
     $ apt-get install locales
     $ dpkg-reconfigure locales
 
+You can resolve this for macOS systems by configuring your shell's locale::
```

Comment:
You'll need to add `macOS` to https://github.com/django/django/blob/master/docs/spelling_wordlist to for spell-check to pass.